doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -20,20 +20,148 @@
+
+<p align="center"><b>Table 2-5:</b> Kubernetes Architecture: Operations and LCM Requirements </p>
+
 
+<a name="2.3.6"></a>
+### 2.3.6 Assurance
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.asr.mon.01` | Integration | The Architecture **must** include integration with infrastructure components to support collection of telemetry for assurance monitoring and network intelligence. |
+| `req.asr.mon.02` | Monitoring | The Architecture **should** support Network Intelligence capabilities that allow richer diagnostic capabilities which take as input broader set of data across the network and from CNF workloads. |
+| `req.asr.mon.03` | Monitoring | The Architecture **must** allow for the collection and dissemination of performance and fault information. |
+| `req.asr.mon.04` | Network | The NFVI Network Fabric and Network Operating System **must** provide network operational visibility through alarming and streaming telemetry services for operational management, engineering planning, troubleshooting, and network performance optimisation. |
+
+<p align="center"><b>Table 2-6:</b> Kubernetes Architecture: Assurance Requirements</p>
+
+<a name="2.3.7"></a>
+### 2.3.7 Security
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.sec.gen.01` | General | The Architecture **should** provide workload isolation. |
+| `req.sec.gen.02` | General | The Architecture **must** support policy based RBAC. |
+| `req.sec.gen.03` | General | The Architecture **must** support a centralised authentication and authorisation mechanism. |
+| `req.sec.gen.04` | General | The Architecture **should** provide Operating System kernel isolation. |
+| `req.sec.gen.05` | General | The Architecture **must** provide compute resources isolation. |
+| `req.sec.gen.06` | General | The Architecture **must** provide storage resources isolation. |
+| `req.sec.gen.07` | General | The Architecture **must** provide network resources isolation. |
+| `req.sec.zon.01` | Zoning | The Architecture **must** support identity management. |
+| `req.sec.zon.02` | Zoning | The Architecture **must** support password encryption. |
+| `req.sec.zon.03` | Zoning | The Architecture **must** support data, at-rest and in-flight, encryption. |
+| `req.sec.zon.04` | Zoning | The Architecture **must** support integration with Corporate Identity Management systems. |
+| `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable standards and regulations. |
```

Comment:
Isn't this an open-ended requirement  *all applicable standards and regulations.* ?

Reply:
Agree' req.sec.cmp.02 is too broad and not measurable or testable.

Reply:
@msidana @kedmison Thanks. Changed 
| `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable national standards and regulations. |
|--------------|--------------|---------------------|
in line with:
| `req.sec.cmp.03` | Compliance | The Architecture **must** comply with all applicable regional standards and regulations. |
|--------------|--------------|---------------------|

Reply:
@pgoyal01 thanks for the revision.  However, I am concerned that this is still not a testable requirement.  What would the explicit list of applicable national requirements be?  What would the list of applicable regional standards?  Can we instead cite directly the applicable standards, as we know them, so that the validation/verification teams know which ones to test against?